libs/sdk-backend-tiger/src/convertors/fromBackend/CatalogConverter.ts

```diff
@@ -45,7 +45,7 @@ const tigerLabelToDisplayFormMd = (label: JsonApiLabelOutWithLinks): IAttributeD
         const labelBuilder = commonMetadataObjectModifications(label)(
             builder,
         ) as AttributeDisplayFormMetadataObjectBuilder;
-        labelBuilder.displayFormType(convertLabelType(label.attributes?.valueType));
+        labelBuilder.displayFormType(convertLabelType(label.attributes?.valueType, label.id));
```

Comment:
This is propagated just for POC purposes to change label type for labels based on identity of the expected labels from "mocked" LDM, i.e., model that has data in expected format but do not support new label types yet.